src/app/main/component/events/components/create-edit-events/create-edit-events.component.ts

```diff
@@ -162,6 +162,12 @@ export class CreateEditEventsComponent extends FormBaseComponent implements OnIn
 
   public checkTab(tag: TagObj): void {
     tag.isActive = !tag.isActive;
+    const tagsArr: Array<string> = this.tags.filter((t) => t.isActive).reduce((ac, cur) => [...ac, cur.nameEn], []);
```

Comment:
Is there any reason to use reduce if we use tagsArr only to count length? After filtration, it will be the same length

Reply:
Yes, I tried to do it shorter too. But it needs to reduce the array because of the interface.

Reply:
So, with this logic you can write just this.checkAfterSend = this.tags.some((t) => t.isActive); S.A. you need to check if there is at least one active item